DSharpPlus/Entities/Emoji/DiscordEmoji.cs

```diff
@@ -69,6 +69,12 @@ public partial class DiscordEmoji : SnowflakeObject, IEquatable<DiscordEmoji>
         [JsonProperty("animated")]
         public bool IsAnimated { get; internal set; }
 
+        /// <summary>
+        /// Gets whether this emoji is a unicode one.
+        /// </summary>
+        [JsonIgnore]
+        public bool IsUnicode => this.Id == 0;
+
```

Comment:
Not a discord property.

Reply:
I can remove it, thought of it as a good idea, but I can do the check directly anyway... But it still boils down to the general code style, and I do need a definitive position on this so I could fix it to adhere.